Feature/rb 8213857642/home page setup

rb-blackberg opened the pull request:
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- Verify all items in the attached image in the comments have been addressed.

## Related Tickets & Documents
[- Monday Task URL: ](https://blackberg-group.monday.com/boards/1931719392/pulses/8428879448)

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _site/assets/img/home/conference-group-tablet.webp
 - _site/assets/img/home/facility-tablet.webp
 - _site/assets/img/home/simlearn-hero-mobile.webp
 - _site/assets/img/home/simlearn-hero-tablet.webp
 - _site/assets/img/home/vr-training-tablet.webp
 - assets/img/home/conference-group-tablet.webp
 - assets/img/home/facility-tablet.webp
 - assets/img/home/simlearn-hero-mobile.webp
 - assets/img/home/simlearn-hero-tablet.webp
 - assets/img/home/vr-training-tablet.webp
 - _data/navigation.js
 - _includes/home/01-hero.liquid
 - _includes/home/02-feature-cta.liquid
 - _includes/home/03-transforming-healthcare.liquid
 - _includes/home/04-what-can-we-do.liquid
 - _includes/home/05-what-is-simlearn.liquid
 - _includes/home/06-im-looking-for.liquid
 - _includes/home/07-programs.liquid
 - _site/about-us/index.html
 - _site/about-us/simlearn-leadership/index.html
 - _site/assets/css/styles.css
 - _site/assets/js/home.js
 - _site/assets/sass/vha/components/button/_index.scss
 - _site/assets/sass/vha/components/navigation/navbar.scss
 - _site/assets/sass/vha/content/animations.scss
 - _site/assets/sass/vha/content/images.scss
 - _site/assets/sass/vha/pages/home/01-hero/_index.scss
 - _site/contact-us/index.html
 - _site/course-catalog/index.html
 - _site/index.html
 - _site/news-and-media/index.html
 - _site/portfolio/index.html
 - assets/css/styles.css
 - assets/js/home.js
 - assets/sass/vha/components/button/_index.scss
 - assets/sass/vha/components/navigation/navbar.scss
 - assets/sass/vha/content/animations.scss
 - assets/sass/vha/content/images.scss
 - assets/sass/vha/pages/home/01-hero/_index.scss

rb-blackberg committed:
fix: styles updates from qc

rb-blackberg committed:
fix: qc updates

rb-blackberg committed:
fix: remove mobile stacking

rb-blackberg committed:
fix: adjust responsive images

rb-blackberg commented:
![2025](https://github.com/user-attachments/assets/c6b93f49-ca35-4366-9f6f-77b561286a70)

I've addressed all items in the provided feedback.

RyanCrux reviewed (changes requested):
I found a couple small things:

<img width="277" alt="Screenshot 2025-02-12 at 1 06 52 PM" src="https://github.com/user-attachments/assets/7d2da076-9ed4-41f0-85ee-f5d61e5298fe
The gradient doesn't fade in or change on hover. This may be exactly the way you and Will discussed but I am mentioning it based on the comment to double check:
<img width="606" alt="Screenshot 2025-02-12 at 1 01 07 PM" src="https://github.com/user-attachments/assets/b55bf13f-505a-4fbc-bffc-aea543518b70" />

Border radius is still showing on this image: 
<img width="434" alt="Screenshot 2025-02-12 at 12 58 19 PM" src="https://github.com/user-attachments/assets/f9b36fc2-6fac-4455-ae07-b6411f8e3336" />

<img width="185" alt="Screenshot 2025-02-12 at 12 57 58 PM" src="https://github.com/user-attachments/assets/1cdc0494-88b2-4ca3-8de6-a410f71afbff" />

rb-blackberg committed:
bug: hero img border radius; button gradient hover state

rb-blackberg commented:
@RyanCrux Thanks for catching that. I've updated the hero image border radius and added a transition to those button backgrounds - the gradient does animate it's just very subtle and only does it once.

will-bbg commented:
@rb-blackberg aw yeah. This is looking great! I'm going to edit the hero image a bit. I'll update the figma file. Stand by.

will-bbg commented:
@rb-blackberg i've updated the figma file. Please note changes in the screenshot below. 

<img width="1063" alt="Screenshot 2025-02-13 at 7 35 51 AM" src="https://github.com/user-attachments/assets/0a425db0-28d0-4e93-b3b6-0669ac3b0e96" />

rb-blackberg committed:
feat: hero image style updates

will-bbg commented:
@rb-blackberg can you confirm that the new image includes the gradients that I highlighted in the previous comment?

rb-blackberg committed:
feat: swap home hero images

rb-blackberg commented:
@will-bbg I've swapped out the home hero images for desktop and tablet

will-bbg commented:
@rb-blackberg This is looking so good! I have just two minor things:

1. Would you please double check that the exported hero image also includes the blurred white elements from the design? I'm including a screenshot: 
![image](https://github.com/user-attachments/assets/e7bb7c35-1bf2-4edd-8d27-9ca2b2f3a04c)

2. I was checking the page on my phone (iphone 15) and found a slight overlap with the grid pattern. Please see the following figma: https://www.figma.com/design/Xbl0fIZz0duCzB51b5I24U/SimLEARN-Website?node-id=1251-2&t=VdbTHPIWnlV1BIOO-1

rb-blackberg committed:
bug: fix grid pattern overflow; export hero images

rb-blackberg commented:
@will-bbg I've made updates for the two items in your feedback:

- exported hero images from the entire group
- fixed grid pattern overflow issue

will-bbg reviewed (approved):
🚀

rb-blackberg committed:
fix: merge conflicts
